tools/sigh.ts

```diff
@@ -629,12 +629,19 @@ function runTests(args: string[]): boolean {
 
 // Watches for file changes, then runs the steps for the first item in args, passing the remaining items.
 function watch(args: string[]): boolean {
+  const options = minimist(args, {
+    string: ['dir'],
+    default: {dir: '.'},
+    stopEarly: true,  // Allow options to be used in the command; 'sigh watch --dir src test -g foo'
+  });
+
   if (chokidar === null) {
     console.log('\nthe sigh watch subcommand requires chokidar to be installed. Please run \'npm install --no-save chokidar\' then try again\n');
     return false;
   }
-  const command = args.shift() || 'webpack';
-  const watcher = chokidar.watch('.', {
+
+  const command = options._.shift() || 'webpack';
```

Comment:
FMI: what is `options._`?

Reply:
It's the tokens remaining in the input args after minimist stops consuming them. So `sigh watch --dir src test -g foo` would give you `{ _: [ 'test', '-g', 'foo' ], dir: 'src' }` as the options object.

Reply:
Gotcha, thanks!